index.html

```diff
@@ -0,0 +1,224 @@
+        <div class="techstack-container">
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="techstack-container">
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="techstack-container">
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="techstack-container">
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="techstack-container">
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="techstack-container">
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+    </section>
+
+    <section class="about">
+      <div class="about-me">
+        <h1>About me</h1>
```

Comment:
- [ ] Please make sure you have only one h1 tag in all HTML files is used only for the make title so please change it, for subtitle use  (e.g., "About myself") uses a header tag `hx( he, h3, h4, h5 ......)` that is for a general convention :+1: